src/ImageProcessor.Web/Caching/IImageCache.cs

```diff
@@ -41,6 +41,16 @@ public interface IImageCache
         int BrowserMaxDays { get; set; }
 
         /// <summary>
+        /// Gets or sets the maximum number folder levels to nest the cached images.
+        /// </summary>
+        int FolderDepth { get; set; }
```

Comment:
Adding to an interface will be a 'breaking' change, would either need to be documented since you aren't releasing a major version, or you could create an IImageCache2 interface to avoid the break